benchmarks/src/main/java/org/apache/druid/benchmark/query/SqlVsNativeBenchmark.java

```diff
@@ -116,12 +115,10 @@ public void setup()
     final PlannerConfig plannerConfig = new PlannerConfig();
 
     this.walker = closer.register(new SpecificSegmentsQuerySegmentWalker(conglomerate).add(dataSegment, index));
-    final DruidSchema druidSchema = CalciteTests.createMockSchema(conglomerate, walker, plannerConfig);
-    final SystemSchema systemSchema = CalciteTests.createMockSystemSchema(druidSchema, walker, plannerConfig);
-
+    final SchemaPlus rootSchema =
```

Comment:
This one is `final` but the one in SqlBenchmark isn't, any reason?

(Fwiw I usually like labeling things final whenever possible but I don't insist on it, except for class fields)

Reply:
I don't usually mark variables in a function as final, but I was trying to follow the pattern I saw in the class. That's why it's inconsistent :\ I'll clean it up

Reply:
In that case, keeping to the style of the surrounding code is probably best. I withdraw my request for consistency across files. (Unless we can reach consensus on whether vars should be marked final whenever possible.)